ursula_cli/shell.py

```diff
@@ -389,9 +390,9 @@ def _run_heat(args, hot):
         while not test_ssh(floating_ip, user, ssh_key_path):
             LOG.debug("waiting for SSH connectivity...")
             time.sleep(5)
-    else:
```

Comment:
why are we killing the else?   without that we'll try to test connecting to the private IPs even if we're trying to use a public floating IP which will fail because the test_ssh doesn't read the ssh_config that tells it to use the proxycommand.